configs/projects/pl-boost.rb

```diff
@@ -14,7 +14,7 @@
   proj.vendor "Puppet Labs <info@puppetlabs.com>"
   proj.homepage "https://www.puppetlabs.com"
 
-  if platform.is_cross_compiled_linux?
+  if platform.is_cross_compiled? || platform.name =~ /solaris-11/
```

Comment:
Why is this needed on Solaris 11?

Reply:
It's due to this in puppet-agent: https://github.com/puppetlabs/puppet-agent/blob/78ff7d8bd6bf5f2e78f3f2e26255bde0238053b9/configs/components/leatherman.rb#L14

I have a ticket (BUILD-107) to clean this up. My goal with this PR was just to get things working with the p-a repo as it was.